tchannel/testing/vcr/patch.py

```diff
@@ -82,27 +82,35 @@ def send(self, arg1, arg2, arg3,
         headers = headers or {}
         headers.setdefault('as', self.arg_scheme)
 
-        vcr_request = VCRProxy.Request(
+        vcr_request = proxy.Request(
             serviceName=self.service.encode('utf-8'),
             hostPort=self.hostport,
-            knownPeers=self.original_tchannel.peers.hosts,
+            knownPeers=[bytes(h) for h in self.original_tchannel.peers.hosts],
             endpoint=endpoint,
             headers=(yield read_full(arg2)),
             body=(yield read_full(arg3)),
-            argScheme=getattr(VCRProxy.ArgScheme, self.arg_scheme.upper()),
+            argScheme=getattr(proxy.ArgScheme, self.arg_scheme.upper()),
             transportHeaders=[
-                VCRProxy.TransportHeader(k, v) for k, v in headers.items()
+                proxy.TransportHeader(bytes(k), bytes(v))
+                for k, v in headers.items()
             ],
         )
 
         # TODO what to do with traceflag, attempt-times, ttl
         # TODO catch protocol errors
 
+        # from tchannel import TChannel
+        from tchannel import TChannel
+        tchannel = TChannel('proxy-client')
+
+        thrift_request = self.vcr_client.send(vcr_request)
```

Comment:
This is not really a client anymore. It's a little confusing to call it that. How about `vcr_service`?